shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -96,8 +98,11 @@ private void load(AlluxioURI filePath, boolean local) throws AlluxioException, I
         load(newPath, local);
       }
     } else {
+      ReadPType readType = mFsContext.getPathConf(filePath)
```

Comment:
Will `getPathConf` incur an RPC each time?

Reply:
changed to hard code `CACHE` so this is avoided